packages/widget-chat/src/components/activity-title-bar/index.test.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {findRenderedDOMComponentWithClass, renderIntoDocument} from 'react-addons-test-utils';
+
+import ActivityTitleBar from '.';
+
+let user;
+let component;
+
+describe(`ActivityTitleBar component`, () => {
```

Comment:
This is the title bar that appears at the top of the chat window, so single activity. It is using the same naming convention as the web client.